src/exercises/cheriabi/README.md

```diff
@@ -37,9 +37,23 @@ associated with a page, `munmap` pages, or even request a replacement mapping
 evasion of CHERI capabilities' protection properties, as CHERI capabilities are
 interpreted in combination with the virtual memory map.
 
-Therefore, the CheriBSD kernel avails itself of a *software permission* bit in
-CHERI capabilities.  Such permissions are not architecturally interpreted but
-are still subject to architectural protection (and so, for example, a zero
+However, in CheriBSD, the kernel can return memory mappings (from `mmap`) using
+a bounded capability.  Moreover, it can demand that references to memory
+mappings passed to other system calls (such as `munmap`, `madvise`, or even
+`mmap` itself) be made using capabilities.  This reduces ambient authority
+significantly: software must have a capability to a part of the memory map
+rather than merely the relevant address.
```

Comment:
This last sentence is slightly loosely phrased -- maybe it would be clearer to write, "... rather than simply passing the integer address"? Or something like that to make it more clear what a contrasting design choice might be.

Reply:
A little caution is required here as you can use an integer address to map at a give location IFF there is no existing reservation in that location.